docs/knowledge_base/tips/skippable_tasks/index.md

```diff
@@ -0,0 +1,208 @@
+
+Taipy is a Python library that's user-friendly and made for creating web applications 
+with interactive interfaces that handle data. 
+This article is all about Taipy's back-end capabilities. 
+Just as a reminder, the main goal of Taipy's back-end is to help you build 
+and manage complex workflows, such as data processing pipelines.
+
+![Skippable Tasks](skippable_tasks.png){width=100%}
+
+When engineers create Directed Acyclic Graphs (DAGs), they often leave out certain tasks. 
+This common practice allows tasks to be orchestrated in a more dynamic and advanced way. 
+One of the key features of Taipy's back-end is skippable tasks, which can be skipped under specific conditions. 
+In this article, we'll explore how skippable tasks work in Taipy and how to use them effectively.
+
+## Setting up Data Nodes for Tasks
+
+A Task in Taipy is a way to represent a Python function that you want to use in the execution graph. 
+
+It contains:
+
+- Input and output Data nodes.
+- The Python function that you've defined and want to use.
+
+![Setting up Data Nodes for Tasks](skippable_tasks_2.png){width=100%}
+
+Before you start using skippable tasks, it's important to configure your tasks correctly 
+with their Data nodes. You can refer to our previous article or 
+documentation for more details on [Data nodes](https://www.taipy.io/tips/the-data-nodes/).
+
+For instance, let's say you have a function like **multiply_and_add()** that takes two parameters 
+and returns two values. How can you represent this function as a Taipy Task?
+
+```python linenums="1"
+def multiply_and_add(nb1, nb2):
+    return nb1 + nb2, nb1 * nb2
+```
+
+In the animation below:
+
+- The first tab corresponds to the creation of the configuration graphically using Taipy Studio.
+- The second tab corresponds to the creation of the very same configuration programmatically.
+
+The order in which you supply Data nodes to the Task is critical. 
+Taipy calls the function using the parameters in the same order as the Data nodes, 
+and the results are returned in that exact order.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/04/full_config.mp4?_=1" frameborder="0" allowfullscreen></iframe>
+
+```python linenums="1"
```

Comment:
When the code is not explained line by line, we don't need to set the line number.

```suggestion
```python
```